scripts/git-extensions.ts

```diff
@@ -0,0 +1,35 @@
+import { SimpleGit } from 'simple-git';
+
+export class GitExtensions {
```

Comment:
does it make more sense to extend SimpleGit rather than wrap it so that consumers can call all of it's methods + these helper methods?